tests/frameworks/test_mlbase.py

```diff
@@ -22,36 +30,76 @@ def get_dataset(classification=True):
 
 
 def run_mlbase_sklearn_classification(context):
-    from mlrun.frameworks.sklearn import apply_mlrun as apply_mlrun_sklearn
-
-    model = LogisticRegression(solver="liblinear")
+    model = LogisticRegression()
     X_train, X_test, y_train, y_test = get_dataset()
-    model = apply_mlrun_sklearn(
-        model, context, model_name="my_model_name", X_test=X_test, y_test=y_test
+    model = apply_mlrun(
+        model, context, X_train=X_train, y_train=y_train, X_test=X_test, y_test=y_test
     )
     model.fit(X_train, y_train)
 
 
 def run_mlbase_xgboost_regression(context):
-    import xgboost as xgb
+    try:
+        import xgboost as xgb
 
-    from mlrun.frameworks.xgboost import apply_mlrun as apply_mlrun_xgb
+        model = xgb.XGBRegressor()
+        X_train, X_test, y_train, y_test = get_dataset(classification=False)
+        model = apply_mlrun(
+            model,
+            context,
+            X_train=X_train,
+            y_train=y_train,
+            X_test=X_test,
+            y_test=y_test,
+        )
+        model.fit(X_train, y_train)
+    except ModuleNotFoundError:
+        pass
 
-    model = xgb.XGBRegressor()
-    X_train, X_test, y_train, y_test = get_dataset(classification=False)
-    model = apply_mlrun_xgb(model, context, X_test=X_test, y_test=y_test)
-    model.fit(X_train, y_train)
+
+def run_mlbase_lgbm_classification(context):
+    try:
+        import lightgbm as lgb
+
+        model = lgb.LGBMClassifier()
+        X_train, X_test, y_train, y_test = get_dataset()
+        model = apply_mlrun(
+            model,
+            context,
+            X_train=X_train,
+            y_train=y_train,
+            X_test=X_test,
+            y_test=y_test,
+        )
+        model.fit(X_train, y_train)
+    except ModuleNotFoundError:
+        pass
 
 
-@pytest.mark.skipif(skip_tests, reason="missing packages")
 def test_run_mlbase_sklearn_classification():
     sklearn_run = new_function().run(handler=run_mlbase_sklearn_classification)
-    model = sklearn_run.artifact("my_model_name").meta
-    assert model.metrics["accuracy"] > 0, "wrong accuracy"
-    assert model.model_file == "LogisticRegression.pkl"
+    assert (sklearn_run.artifact("model").meta.to_dict()["metrics"]["accuracy"]) > 0
+    assert (
+        sklearn_run.artifact("model").meta.to_dict()["model_file"]
+    ) == "LogisticRegression.pkl"
 
 
-@pytest.mark.skipif(skip_tests, reason="missing packages")
+@pytest.mark.skipif(_is_installed("xgboost"), reason="missing packages")
```

Comment:
be more specific, same for other tests
```suggestion
@pytest.mark.skipif(_is_installed("xgboost"), reason="xgboost package missing")
```